### PR TITLE
DOC: Updating notebook

### PR DESCRIPTION
**Pull request by mortonjt**
The following is done in the pull request

1. The ordering of commands is changed in the 88 soils tutorial in order to properly generate the dendrogram heatmaps

2. The last plot in the cf study notebook is left for an exercise.  The real issue can be found here: https://github.com/biocore/gneiss/issues/136

### CODE REVIEW

**Comment by mortonjt** [comment]
Also note that these notebooks are ready for review.

File: ipynb/cfstudy/cfstudy.ipynb [hunk 1]
```diff
@@ -668,7 +668,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We'll plot the how the balances change with respect to the pH.  Each patient will have their own unique color."
+    "We'll plot the how top balance change with respect to the pH.  Each patient will have their own unique color."
```
Comment (antgonza) [inline ipynb/cfstudy/cfstudy.ipynb]: Can you double check this text?

Comment (mortonjt) [inline ipynb/cfstudy/cfstudy.ipynb]: That's a bad typo.  Thanks for catching!

**Comment by mortonjt** [comment]
Thanks!

**Pull request merged**